src/components/generics/vacation-test-forms/employee-vacation-requests.tsx

```diff
@@ -68,17 +72,23 @@ const RenderEmployeeVacationRequests = ({ persons }: Props) => {
   const { accessToken } = useAppSelector(selectAuth);
   const context = useContext(ErrorContext);
   const [ requests, setRequests ] = useState<VacationRequest[]>([]);
+  const [ statuses, setStatuses ] = useState<VacationRequestStatus[]>([]);
+  const [ latestRequestStatuses, setLatestStatuses ] = useState<VacationRequestStatus[]>([]);
   const [ openRows, setOpenRows ] = useState<boolean[]>([]);
   const [sortBy, setSortBy] = useState<VacationRequestSort>(VacationRequestSort.START_DATE);
   const [sortOrder, setSortOrder] = useState<"asc" | "desc">("asc");
+  const [ loading, setLoading ] = useState<boolean>(false);
+  const [ approvalChanged, setApprovalChanged ] = useState<boolean>(false);
+  const statusTextReferenceObject = useRef<any>();
```

Comment:
Where possible we try to avoid the use of any in TS, can this be typed more specifically?

Reply:
Added an appropriate type
```suggestion
  const statusTextReferenceObject = useRef<HTMLParagraphElement>();
``` type